javascript/app-event-analysis/src/components/Grid.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import React, { memo, useMemo } from 'react'
+
+import { useLineColors } from '../utils/useColors.js'
+import type { GridProps } from './Grid.types.js'
+import { GridLine } from './GridLine.js'
+import { GridLineType } from './GridLine.types.js'
+
+export const Grid: React.FC<GridProps> = memo(function Grid({
+	height,
+	width,
+	xScale,
+	yScale,
+}) {
+	const colors = useLineColors()
+	const transform = useMemo<string>(() => `translate(0, ${height})`, [height])
+
+	return (
+		<>
+			<GridLine
+				type={GridLineType.Vertical}
+				myscale={yScale}
+				tickSize={width}
+				ticks={10}
+				color={colors.gridLine}
+				opacity={0.33}
```

Comment:
thematic does actually include stroke opacity in the config. The default is 1.0 so it may be darker than we want here, but if so that might mean we should update thematic.

So you can do something like:

```
const line = theme.gridLines()

<GridLine
color={line.stroke().hex()}
opacity={line.strokeOpacity()}
/>
```